homework/shelkovich_ilya/arrayService/src/test/java/com/tutrit/java/quickstart/service/ArrayServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ArrayServiceTest {
+
+    ArrayService arrayService;
+
+    @Before
+    public void setUp() {
+        this.arrayService = new ArrayService();
+    }
+
+    @Test
+    public void createArray() {
+        Assert.assertArrayEquals(new int[]{}, arrayService.createArray(0));
+    }
+
+    // не уверен что этот тест проверяет то, что я хотел. Я хотел проверить заполняется ли массив случайными числами
+    @Test
+    public void fillRandomly() {
+        int[] array = new int[10];
+        Assert.assertArrayEquals(array, arrayService.fillRandomly(array));
```

Comment:
You check reference on object.
The test will "true" always.
You need check value in massive maybe.

Reply:
don't we have an assert.arrayNotEmpty? I guess something like that should be.

Reply:
I don't have this kind of assertion

Reply:
fixed